views/confirm-password.html

```diff
@@ -129,6 +129,12 @@ <h3 id="error-text">Password updated successfully</h3>
 		theSuccessMessage.style.display = 'none';
 		theErrorsMessage.style.display = 'none';
 
+        if(document.getElementById('password').value.length < 8 ||  document.getElementById('password').value.length > 15){
```

Comment:
Again, length limits should be accepted as inputs. Shouldn't be hard-coded, different people may want to use different limits.